gravity-forms/gw-coupons-exclude-products.php

```diff
@@ -19,10 +19,11 @@ public function __construct( $args ) {
 
 		// set our default arguments, parse against the provided arguments, and store for use throughout the class
 		$this->_args = wp_parse_args( $args, array(
-			'form_id'                => false,
-			'exclude_fields'         => array(),
-			'exclude_fields_by_form' => array(),
-			'skip_for_100_percent'   => false,
+			'form_id'                        => false,
+			'exclude_fields'                 => array(),
+			'exclude_fields_without_options' => array(),
```

Comment:
Since this is about excluding the _options_ and not the field itself, it might make more sense to call this: (or something similar)

```suggestion
			'exclude_options_from_fields' => array(),
```